app/Http/Controllers/API/GroupController.php

```diff
@@ -867,12 +868,29 @@ public function createGroupv2(Request $request): JsonResponse {
             $file->upload('image', 'image', $idGroup, env('TBL_GROUPS'), false, true, true);
         }
 
-        // Notify relevant admins.
-        $notify_admins = Fixometer::usersWhoHavePreference('admin-moderate-group');
-        Notification::send($notify_admins, new AdminModerationGroup([
-                                                                        'group_name' => $name,
-                                                                        'group_url' => url('/group/edit/'.$idGroup),
-                                                                    ]));
+        // Check if groups should be auto-approved
+        if (env('FEATURE__AUTO_APPROVE_GROUPS', false)) {
+            // Auto-approve the group
+            $group->update(['approved' => true]);
+            
+            // Fire the approval event
+            event(new \App\Events\ApproveGroup($group, $data));
+            
+            // Notify the creator that their group was approved
+            Notification::send($user, new \App\Notifications\GroupConfirmed([
+                'group_name' => $name,
+                'group_url' => url('/group/view/'.$idGroup),
+            ]));
+            
+            Log::info("Auto-approved group: $idGroup");
```

Comment:
Is this stray? Or did you mean to leave this in?

Reply:
I am just going to leave it for now.